doc/migrations/4.0.1_4.x.x.md

```diff
@@ -6,3 +6,30 @@ Currently, only the `urn:xmpp:http:upload:0` XMLNS is served.
 All major, modern client libraries and applications support the 0.3.0+ specification.
 If you experience any issues with making requests to the HTTP File Upload service, please update your client.
 
+## Retirement of the old `*.cfg` format
+
+Since release 4.x.x, we are no longer supporting `*.cfg` MongooseIM configuration format. For further application's usage, please use `TOML` format.
```

Comment:
something is wrong about this sentence. I would definitely say `the cfg format` and `the TOML format` - with the definite articles. But the `futher application's usage` is also stylistically awkward, I found zero matches in Google so I am most likely correct here. Could you rephrase this, e.g. just "Please use the TOML format instead."?